### PR TITLE
[typo](docs) optimization Monitoring and alarming doc

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

File: docs/en/docs/admin-manual/maint-monitor/monitor-alert.md [hunk 1]
```diff
@@ -63,42 +63,43 @@ Doris's monitoring data is exposed through the HTTP interface of Frontend and Ba
...
-    jvm_young_gc{type="time"} 93650
-    # HELP  jvm_old_gc jvm old gc stat
-    # TYPE  jvm_old_gc gauge
-    jvm_old_gc{type="count"} 21
-    jvm_old_gc{type="time"} 58268
-    # HELP  jvm_thread jvm thread stat
-    # TYPE  jvm_thread gauge
-    jvm_thread{type="count"} 767
-    jvm_thread{type="peak_count"} 831
+      # HELP  jvm_heap_size_bytes jvm heap stat
```
Comment (morningman) [inline docs/en/docs/admin-manual/maint-monitor/monitor-alert.md:66]: There should no space in front of

**Comment by zy-kkk** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**